pkg/kubelet/container/runtime.go

```diff
@@ -94,6 +94,8 @@ type Runtime interface {
 	RemoveImage(image ImageSpec) error
 	// Returns Image statistics.
 	ImageStats() (*ImageStats, error)
+	// Returns the filesystem path of the pod's network namespace
+	GetNetNS(containerID ContainerID) (string, error)
```

Comment:
> Is it essential for adding GetNetNS() to Runtime interface? Not all container runtimes (e.g., HyperContainer, which is a hypervisor-based container runtime) run containers inside network namespace.

@feiskyer Good point, though I'd like to reduce the usage of casting in the network plugins and potentially make them usable for more runtimes.  What if we specified in the Runtime interface that a Runtime was not required to return a valid netns path if that runtime does not do namespace creation?  Network plugins would then need to check if the returned path was "" and return an error if they could not continue without a path.